pokedex2.0/src/CharacterItem.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+export default Class haracterItem extends React.Component {
```

Comment:
Class needs a little c in order to work. You also forgot the c for CharacterItem